backends/server/routes/userRoutes.ts

```diff
@@ -0,0 +1,73 @@
+import express, { Router, Request, Response } from "express";
+import { PrismaClient } from "@prisma/client";
+
+const router = Router();
+const prisma = new PrismaClient();
+
+// Create a new user
+router.post("/", async (req: Request, res: Response) => {
+  try {
+    const { firstName, lastName, email, password, address, phoneNumber } =
+      req.body;
+    const newUser = await prisma.user.create({
+      data: { firstName, lastName, email, password, address, phoneNumber, createdAt: new Date(), updatedAt: new Date() },
+    });
```

Comment:
[Should]
Every time you will add a field to the `User` you will need to add it here also.
This is a possible future bug.
```suggestion
    const newUser = await prisma.user.create({
      data: req.body,
    });
```

take notice you have a code-smell in your code that should trigger this kinf of thinking.
```                     const { firstName, lastName, email, password, address, phoneNumber } = req.body;```
```prisma.user.create({ data: { firstName, lastName, email, password, address, phoneNumber, createdAt: new Date(), updatedAt: new Date() } });```

Those lines are very similar and should trigger a thought: "Is it really necessary writing this twice?"